mods/extrahorns/init.lua

```diff
@@ -0,0 +1,66 @@
+-- large horn
+minetest.register_craftitem ("extrahorns:largehorn", {
+	description = "Large Horn",
+	inventory_image = "extrahorns_largehorn.png",
+	groups = {instrument=1},	
+	on_use = function (itemstack, user)
+		minetest.sound_play ("extrahorns_largehorn", {
+			pos = user:getpos(),
+			max_hear_distance = 450,
+			gain = 1,
+		})
+	end,
+})
+
+minetest.register_craft ({

```

Comment:
Space between function name and `(` isn't required, but is up to you to keep it or remove it.